src/Paket.Bootstrapper/HelperProxies/WebRequestProxy.cs

```diff
@@ -7,6 +7,7 @@ public class WebRequestProxy : IWebRequestProxy
     {
         public WebRequestProxy()
         {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
Suggestion to enable TLS1.2 while leaving other enabled security protocols untouched via:
`ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls12;`

Reply:
I would say do this


```
ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12
                                         | SecurityProtocolType.Tls11
                                         | SecurityProtocolType.Tls
                                         | SecurityProtocolType.Ssl3;
```